website/docs/r/datacenter.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_datacenter"
+sidebar_current: "docs-vsphere-resource-datacenter"
+description: |-
+  Provides a VMware vSphere datacenter resource. This can be used as the primary container of inventory objects such as hosts and virtual machines.
+---
+
+# vsphere\_datacenter
+
+Provides a VMware vSphere datacenter resource. This can be used as the primary container of inventory objects such as hosts and virtual machines.
+
+No updates are possible.
```

Comment:
I added the note you mentioned. In the beginning I put it like that more than anything to show that as implemented here there can be no updates, but I agree that it wasn't clear enough. Saying that it can't be updated is a small lie though, as far as I know there is at least one case in which an update within vSphere to a datacenter would be possible, which would be moving the datacenter to a different folder. 

So far I haven't implemented update but will look into it. I would like to finish two other resources I'm working on (distributed virtual switches and distributed portgroups) and then come back to this if that's ok.